### ISSUE TITLE
Firefox plugin does not work

### ISSUE THREAD

**Issue by MildlyInterested** [qazxcdswe123/search-obsidian-in-google#4]
1. Installed addon from the firefox store (https://addons.mozilla.org/en-US/firefox/addon/search-obsidian-in-google/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)
2. Allowed access to google.com in addon setting
![image](https://github.com/qazxcdswe123/search-obsidian-in-google/assets/57712666/437cb87b-9e3a-443e-a6c8-a9784a0c3ab0)
3. I've tried the same on a fresh install on chrome (it worked there) and a fresh install of firefox (didn't work) to rule out any obsidian issues.

**Comment by qazxcdswe123** [comment]
Yes I've also noticed that but sadly I can't figure out what's the cause of the problem. :(

As the chrome extension works just fine for me, I currently have no idea how or motivation to make it works on firefox, sorry for that.

Maybe someone can help me out? I'll be happy to merge their patch.
